### PR TITLE
Adding blog post for Multimodal semantic search

### PR DESCRIPTION
**Pull request by martin-gaievski**
### Description
Multimodal semantic search blog.
 
### Issues Resolved
https://github.com/opensearch-project/project-website/issues/2380

### Check List
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _community_members/haoyng.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      url: /community/members/index.html
+    - title: 'Hao Yang&apos;s Profile'
+      url: '/community/members/hao-yang.html'
+photo: '/assets/media/community/members/haoyng.jpg'
+job_title_and_company: 'Principal Scientist at Amazon Artificial General Intelligence'
+personas:
+  - author
+permalink: '/community/members/hao-yang.html'
+---
+**Hao Yang** is a Principal Scientist at Amazon Artificial General Intelligence, working on multiple aspects in multi-modal generative models. His own research interests lie in how to efficient and effectively train and adapt large scale computer vision and multi-modal models.
```
Comment (martin-gaievski) [inline _community_members/haoyng.md:21]: `[OpenSearch.Spelling] Error` can be ignored, as this is cause by Hao's name.

File: _posts/2024-03-14-multimodal-semantic-search.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+layout: post
+title:  "Multimodal search: Searching with semantic and visual understanding"
+authors:
+  - gaievski
+  - haoyng
+  - dylantong
+date: 2024-03-24
+categories:
+  - technical-posts
+meta_keywords: Multimodal search, searching with semantic and visual understanding, improve search relevance, hybrid search in OpenSearch 2.10, keyword and image search
```
Comment (pajuric) [inline _posts/2024-03-14-multimodal-semantic-search.md:11]: @martin-gaievski - Please update the meta with the following: 

**Meta keywords:** neural query, multimodal search, search technology, search relevancy, OpenSearch 2.11

**Meta_description:** Learn about the enhancements to OpenSearch 2.11's neural query search by image capabilities, and dive deeper by exploring how multimodal models generate embeddings.


And please update the publish date to today, 4/11/24.

Comment (martin-gaievski) [reply _posts/2024-03-14-multimodal-semantic-search.md]: @pajuric sure, just pushed new commit with requested changes

**Comment by pajuric** [comment]
@nateynateynate @krisfreedain - this blog is ready to push live today. Thanks, guys.

**Review by krisfreedain — DISMISSED** [review]
@martin-gaievski - there's an issue with one of the links - notes in the PR

File: _posts/2024-04-11-multimodal-semantic-search.md [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+
+We realized that some multimodal models can’t operate within these constraints. However, we have broader plans to rework our framework, removing the current limitations and providing more generic custom model integration support. We revealed this plan in a recent [blog post](http://earlier%20this%20year,%20we%20published%20a%20blog%20that%20revealed%20our%202024%20plans%20to%20revamp%20the%20neural%20search%20experience%20and%20deliver%20greater%20flexibility.%20our%20goal%20is%20to%20be%20able%20to%20support%20any%20multi-modal%20model/). Nonetheless, we opted to deliver multi-modal support for neural search sooner because we believe our users have a lot to gain now instead of waiting later this year.
```
[the remark this answers is not in the document]
Comment (martin-gaievski) [reply _posts/2024-04-11-multimodal-semantic-search.md:76]: @dylan-tong-aws Can you please check what's the url we want to use here. As per history of edits you've added that link.

[the remark this answers is not in the document]
Comment (pajuric) [reply _posts/2024-04-11-multimodal-semantic-search.md:76]: @dylan-tong-aws - Is this the blog post you are referencing? https://opensearch.org/blog/opensearch-ai-retrospective/

**Review by nateynateynate — DISMISSED** [review]
Looks and reads awesome. Thanks for your contribution.

File: _posts/2024-04-11-multimodal-semantic-search.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (martin-gaievski) [reply _posts/2024-04-11-multimodal-semantic-search.md]: Just got confirmation from Dylan offline that link suppose to refer https://opensearch.org/blog/opensearch-ai-retrospective/, as we expected. Pushed commit that corrected this

**Review by nateynateynate — APPROVED** [review]
Looks good! Thanks!

**Pull request merged**